### PR TITLE
Fix #352

### LINKED ISSUE #352: "Percentage imputed" is displaying twice when running example_classification_1.py random forest
@mxlei01 Can you take a look at this? I suspect that it isn't your code, but that your code is exposing some other repetition...

## STR
- [ ] modify `example_classification_1.py` to train a random forest model
- [ ] run the example
- [ ] note the output
- [ ] Note this is not an issue in `example_advanced.py`

```
...
3                   4      10002          187.0   135.0     4.4         M   
4                   5      10002          188.0   125.0     4.3         M   

  ThirtyDayReadmitFLG  
0                   N  
1                   N  
2                   N  
3                   N  
4                   N  
Percentage Imputed: 1.3171225937183384%
Percentage Imputed: 1.3171225937183384%

Training Random Forest Classification
RandomForestClassifier Training Results:
- Training time:
...
```

## Additional notes

- It would be nice to round this to 2 or three decimals

### PR DESCRIPTION
**Pull request by mxlei01**
Resolves #352, added a parameter to output prints from imputes value.

User is warned that imputation always occurs at prediction, even though the user might set `impute=False` for `pipeline`.

### CODE REVIEW

**Comment by mxlei01** [comment]
Done, please have a look.

File: healthcareai/supervised_model_trainer.py [hunk 1]
```diff
@@ -17,8 +17,8 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
             dataframe (pandas.core.frame.DataFrame): The training data in a pandas dataframe
             predicted_column (str): The name of the prediction column 
             model_type (str): the trainer type - 'classification' or 'regression'
-            impute (bool): True to impute data (mean of numeric columns and mode of categorical ones). False to drop rows
-                that contain any null values.
+            impute (bool): True to impute data (mean of numeric columns and mode of categorical ones). False to drop
+            rows that contain any null values.
             grain_column (str): The name of the grain column
             verbose (bool): Set to true for verbose output. Defaults to False.
```
Comment (Aylr) [inline healthcareai/supervised_model_trainer.py]: This docstring doesn't agree with the current default of True.

File: healthcareai/supervised_model_trainer.py [hunk 2]
```diff
@@ -28,8 +28,10 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
         # Build the pipeline
         # Note: Missing numeric values are imputed in prediction. If we don't impute, then some rows on the prediction
         # data frame will be removed, which results in missing predictions.
-        pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=impute)
-        prediction_pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=True)
+        pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=impute,
```
Comment (Aylr) [inline healthcareai/supervised_model_trainer.py]: Here verbose should be True to tell the user what is going on during training

File: healthcareai/supervised_model_trainer.py [hunk 3]
```diff
@@ -28,8 +28,10 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
...
+                                                verbose=False)
+        prediction_pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=True,
```
Comment (Aylr) [inline healthcareai/supervised_model_trainer.py]: Here it should be False.

File: healthcareai/supervised_model_trainer.py (hunk 2 above)
Comment (mxlei01) [reply healthcareai/supervised_model_trainer.py]: Training verbose is now set to True.

File: healthcareai/supervised_model_trainer.py (hunk 1 above)
Comment (mxlei01) [reply healthcareai/supervised_model_trainer.py]: Done, place the default to True.

**Pull request merged**